README.md

```diff
@@ -268,67 +268,66 @@ curl -s http://0.0.0.0:8080/blocks/head | jq
 
-2. Ensure we have the latest polkadot-js dependencies
+1. Ensure we have the latest polkadot-js dependencies. Note: Every monday the polkadot-js ecosystem will usually come out with a new release. It's important that we keep up, and read the release notes for any breaking changes, or high priority updates. You can use the following command `yarn upgrade-interactive` to find and update all available releases. Feel free to update other packages that are available for upgrade if reasonable. To upgrade just `@polkadot` scoped deps use `yarn up "@polkadot/*"`
 
-  * @polkadot/api [release notes](https://github.com/polkadot-js/api/releases)
-  * @polkadot/apps-config [release notes](https://github.com/polkadot-js/apps/releases)
-  * @polkadot/util-crypto [release notes](https://github.com/polkadot-js/common/releases)
+    * @polkadot/api [release notes](https://github.com/polkadot-js/api/releases)
+    * @polkadot/apps-config [release notes](https://github.com/polkadot-js/apps/releases)
+    * @polkadot/util-crypto [release notes](https://github.com/polkadot-js/common/releases)
 
-  Note: Every monday the polkadot-js ecosystem will usually come out with a new release. It's important that we keep up, and read the release notes for any breaking changes, or high priority updates. You can use the following command `yarn upgrade-interactive` to find and update all available releases. Feel free to update other packages that are available for upgrade if reasonable. 
+1. After updating the dependencies, the next step is making sure the release will work against all noted runtimes for Polkadot, Kusama, and Westend. This can be handled via the [sidecar-runtime-test](https://github.com/TarikGul/sidecar-runtime-test) helper library. Instructions for how to run it are in the repos README.md. Before moving forward ensure all tests pass, and if it warns of any missing types feel free to make an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues).
 
-3. After updating the dependencies, the next step is making sure the release will work against all noted runtimes for Polkadot, Kusama, and Westend. This can be handled via the [sidecar-runtime-test](https://github.com/TarikGul/sidecar-runtime-test) helper library. Instructions for how to run it are in the repos README.md. Before moving forward ensure all tests pass, and if it warns of any missing types feel free to make an issue [here](https://github.com/paritytech/substrate-api-sidecar/issues). 
+1. Update the version in the package.json (this is very important for releasing on NPM).
 
-4. Update the version in the package.json (this is very important for releasing on NPM).
+1. Update `CHANGELOG.md` by looking at merged PRs since the last release. Follow the format of previous releases.Only record dep updates if they reflect type definition updates as those affect the users API.
```

Comment:
```suggestion
1. Update `CHANGELOG.md` by looking at merged PRs since the last release. Follow the format of previous releases. Only record dep updates if they reflect type definition updates as those affect the users API.
```